src/core/environment_test.go

```diff
@@ -17,6 +17,38 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
+const lscpuInfo = `
+Architecture:                    aarch64
+CPU op-mode(s):                  32-bit, 64-bit
+Byte Order:                      Little Endian
+CPU(s):                          2
+On-line CPU(s) list:             0,1
+Thread(s) per core:              1
+Core(s) per socket:              2
+Socket(s):                       1
+NUMA node(s):                    1
+Vendor ID:                       ARM
+Model:                           3
+Model name:                      Cortex-A72
+Stepping:                        r0p3
+BogoMIPS:                        166.66
+L1d cache:                       64 KiB
+L1i cache:                       96 KiB
```

Comment:
our code is also trimming quotes, maybe add an example in here to prove that works too?
```suggestion
L1i cache:                       "96 KiB"
```

Reply:
Added tests.